libs/platform/experience/src/routes.ts

```diff
@@ -7,6 +7,11 @@ export const defaultExperienceRoutes: RouteConfig[] = [
   {
     pattern: new URLPattern({ pathname: '/{index.html}?' }),
   },
+  {
+    path: '/login',
+    type: RouteType.Login,
+    redirect: { when: 'USER.AUTHENTICATED' },
```

Comment:
Lets use enter guard instead

Reply:
Also lets move this route into authenticated package

Reply:
`enter` hook came from the source of lit-router implementation and its aim like `beforeMount` (execute some code before route fully processed). It does not make any redirection under the hood. I would suggest to keep it as is and implement separate `redirect` method.

Reply:
But I don't see any reason why we need this redirect method. We already have custom implementation of enter function let's adjust it for our needs.